velero-csi-plugin/csi_restorer_test.go

```diff
@@ -90,33 +99,31 @@ func TestResetPVCAnnotations(t *testing.T) {
 				},
 			},
 			preserveAnnotations: []string{"ann1", "ann2", "ann3", "ann4"},
+			expectedAnnotations: map[string]string{
+				"ann1": "ann1-val",
+				"ann2": "ann2-val",
+				"ann3": "ann3-val",
+				"ann4": "ann4-val",
+			},
+			expectEmptyAnnotations: false,
 		},
 	}
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			resetPVCAnnotations(&tc.pvc, tc.preserveAnnotations)
-			if tc.pvc.Annotations == nil {
-				t.Errorf("%s failed, unexpected PVC annotations returned, Want: non-nil; Got: nil", tc.name)
-			}
-			if tc.expectNonEmptyAnnotations && len(tc.pvc.Annotations) != 0 {
-				t.Errorf("%s failed, unexpected count of PVC annotations returned, Want: 0; Got: %d", tc.name, len(tc.pvc.Annotations))
-			}
-
-			if tc.expectNonEmptyAnnotations {
+			if tc.expectEmptyAnnotations {
```

Comment:
OK, so now that I'm looking at this more closely, I think we can remove L115-126 and replace it with `assert.Equal(t, tc.expectedAnnotations, tc.pvc.Annotations)`, right? 

If so, then can also remove the `expectEmptyAnnotations` field from the anonymous test case struct.

Reply:
Lol! Yes. That would def simplify this!
Updated the tests. Thanks for catching this!